packages/block-editor/src/components/list-view/style.scss

```diff
@@ -310,30 +310,40 @@
 			width: 100%;
 			transform: translateY(-50%);
 		}
+
+		/* Match input's text position with non-editable visual state */
+		.components-input-control {
+			height: 24px;
+			position: relative;
+			left: -8px;
+			border-radius: $radius-block-ui * 2;
+			overflow: hidden; // apply the border radius visually
```

Comment:
The comment is about border but follows overflow. If they're connected the comment could say why.

Reply:
Can't replicate the reason I added this. Removing.